apps/api/docs/development-setup.md

```diff
@@ -11,6 +11,8 @@ Before setting up OsmoX for development, ensure you have the following prerequis
 - **Git:** Git v2.x or higher.
 - **MariaDB:** MariaDB v10.x or higher.
 - **Redis:** Redis v6.x or higher
+- **Docker:** 26.1.2, build 211e74b
```

Comment:
Do we have docker setup instructions even in development setup document? If not, we can remove from this doc.

Also can preferably change the `**Docker:** 26.1.2, build 211e74b` line to `**Docker:** Docker v26.1.2`. Don't need to specify the build, version should be enough.
Let's keep consistency with other pre-requisites mentioned here too: `**Docker Compose:** v2.1.1`->`**Docker Compose:** Docker Compose v2.1.1`

Also, I believe we have docker setup instructions in the portal README. So please add these details there as well.

Reply:
Can further update the PR title accordingly as we are adding more than just provider dependency versions

Reply:
Done, updated the documents with the above suggestions.